.github/workflows/nightly.yml

```diff
@@ -58,3 +59,51 @@ jobs:
     - name: Fuzz long test
       working-directory: ${{github.workspace}}/build
       run: ctest -C ${{matrix.build_type}} --output-on-failure -L "fuzz-long"
+
+  adapter-test:
+    name: Test - Adapters on Ubuntu
+    strategy:
+      matrix:
+        adapter: [L0]
+        build_type: [Debug, Release]
+        compiler: [{c: gcc, cxx: g++}]
+
+    runs-on: devcloud-pvc
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install pip packages
+      run: python3 -m pip install -r third_party/requirements.txt
+
+    - name: Download DPC++
+      run: |
+        wget -O ${{github.workspace}}/dpcpp_compiler.tar.gz https://github.com/intel/llvm/releases/download/sycl-nightly%2F20230626/dpcpp-compiler.tar.gz
+        tar -xvf ${{github.workspace}}/dpcpp_compiler.tar.gz
+
+    - name: Setup DPC++
```

Comment:
is this the only step that's different compared to regular cts runs? If so, maybe it'd be easier to parametrize an existing workflow?

Reply:
I have changed my approach to testing.